documentation/docs/25-build-and-deploy/70-adapter-cloudflare-workers.md

```diff
@@ -72,11 +72,26 @@ import adapter from '@sveltejs/adapter-cloudflare-workers';
 
 export default {
 	kit: {
-		adapter: adapter({ config: '<your-wrangler-name>.toml' })
+		adapter: adapter({
+			config: '<your-wrangler-name>.toml',
+			external: [ 'fs' ]
+		})
 	}
 };
 ```
 
+### Custom config
```

Comment:
Should I change this one to `config` as well, like:

```suggestion
### `config`
```

That would be consistent with the other file, IMO.